typespec-tests/src/main/java/com/_specs_/azure/clientgenerator/core/access/implementation/models/package-info.java

```diff
@@ -2,5 +2,10 @@
 // Licensed under the MIT License.
 // Code generated by Microsoft (R) TypeSpec Code Generator.
 
-/** Package containing the data models for Access. Test for internal decorator. */
+/**
+ *
+ * <!-- start generated doc -->
+ * Package containing the data models for Access. Test for internal decorator.
+ * <!-- end generated doc -->
+ */
```

Comment:
Welcome advise from @srnagar and @haolingdong-msft (it seems not possible for `@Generated` approach here)

Also may want to consider whether we'd like to enable javadoc customization to APIs in Client as well.